### PR TITLE
[IMP] base: rename action multi tool-tip and remove from report - SEB

### PR DESCRIPTION
**Pull request by seb-odoo**
Task: https://www.odoo.com/web#id=1819329&active_id=1819329&view_type=form&model=project.task

### CODE REVIEW

**Comment by tde-banana-odoo** [comment]
@xmo-odoo I bet you love that one.

**Comment by xmo-odoo** [comment]
@tde-banana-odoo #24738

File: odoo/addons/base/models/ir_actions.py [hunk 1]
```diff
@@ -268,7 +268,7 @@ class IrActionsActWindowView(models.Model):
     view_id = fields.Many2one('ir.ui.view', string='View')
     view_mode = fields.Selection(VIEW_TYPES, string='View Type', required=True)
     act_window_id = fields.Many2one('ir.actions.act_window', string='Action', ondelete='cascade')
-    multi = fields.Boolean(string='On Multiple Doc.', help="If set to true, the action will not be displayed on the right toolbar of a form view.")
+    multi = fields.Boolean(string='Restrict to lists', help="If checked and the action is bound to a model, it will only appear in the More menu on list views")
```
Comment (seb-odoo) [inline odoo/addons/base/models/ir_actions.py]: try to remove this one

Comment (seb-odoo) [reply odoo/addons/base/models/ir_actions.py]: is a dependency of view mode? (= not form)

**Pull request closed without merging**